server/dto/service_quality_response.go

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2017 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package dto
+
+// ServiceQualityResponse represents response from the quality oracle service
+type ServiceQualityResponse struct {
+	Connects []QualityConnects `json:"connects"`
+}
+
+// QualityConnects represents a single proposal connection quality description
+type QualityConnects struct {
+	Proposal     QualityProposal `json:"proposal"`
```

Comment:
Why service definition is here. It should not. MORQA service should have just `proposalID` primary key to identity proposal.

Reply:
Proposal ID (current):
```providerID: 0x2```


Proposal ID (implementing now):
```
proposalID: {
  providerID: 0x2,
  serviceType: "openvpn"
}
```

Proposal ID (future):
```
proposalID: {
  providerID: 0x2,
  serviceType: "openvpn",
  serialNumber: 2
}
```

Reply:
Removed definition of these structs.